docs/Reference/API-Methods.md

```diff
@@ -2266,6 +2266,9 @@ You can interact with contracts using [`eth_sendRawTransaction` or `eth_call`].
 
 #### Parameters
 
+When using `eth_call`, by default, the transaction will not fail if there is an insufficient balance in the sender account.
```

Comment:
I wonder if it's even useful to have this here now that you merged the Transaction call object description table.
Or maybe just as a note to say it's new? And then maybe it should be placed after the 2 params descriptions.

Reply:
Comment from @shemnon at https://github.com/hyperledger/besu/pull/1834#issuecomment-768022921 may be interesting to add in the note. I think explaining how the default for call enables the balance to be always ok by using a very big balance is interesting as an additional info:

> When using `eth_call` the balance of the from account does not cause the transaction to fail for lack of balance. This is done by setting the balance of the account to a very large amount of ether. If you want the transaction to fail for insufficient funds you can add `"strict": true` to the call parameters.